### DISCUSSION TITLE
Consiglio tapparelle vimar

### DISCUSSION THREAD

**Discussion by paolinor88** [indomus/forum#667] [Hardware & Firmware]
Ciao a tutti, l’ultimo (per adesso…) tassello mancante del mio puzzle domotico sono le tapparelle; l’impianto esistente è “vimar by-me” che, da quel che ho capito informandomi dal produttore, è una via di mezzo tra una installazione base e un impianto domotico propriamente detto.
Quello che vorrei capire, partendo dalla mia configurazione attuale che allego in foto, è quali componenti devo sostituire e/o integrare per poter controllare le tapparelle con HA.
Grazie a chi saprà aiutarmi..
![696756A1-47A9-428D-9140-3A7FC115A587](https://user-images.githubusercontent.com/68273270/131566016-4f30dcce-0475-4c39-8a0c-05127cab5e7f.jpeg)
![D2C9EB4C-7416-44ED-A9AD-F1317BB1DF0F](https://user-images.githubusercontent.com/68273270/131566020-a6c46243-cc1f-423d-863c-383df18d381d.jpeg)

**Comment by essegi65** [comment]
Non conosco i componenti vimar, ma a vedere la foto con l'attuatore sembra che sia quello da sostituire. Con uno shelly 2.5, che dovrai alimentare, puoi inserire i cavi che attualmente sono collegati a: freccia su, C e freccia giù nei poli dello shelly (trovi gli schemi di collegamento anche su indomus.it). Così facendo perderai le funzioni dell'app vimar (se la usi).

**Comment by paolinor88** [comment]
Intanto grazie per la tua risposta.
Usare uno Shelly 2.5 è stata la prima idea che mi è venuta, avendo già altri Shelly ho dimestichezza con prodotto e configurazione.. ma:
- andrebbe messo “in cascata” e non in sostituzione dell’attuatore, perché altrimenti perderei la comodità del bus che mi consente di azionare tutte le tapparelle contemporaneamente da un comando centrale. E non esistono (o almeno non ho trovato) schemi a riguardo..
- il costo mi frena un po’. 45€ a Shelly x 7 tapparelle…… con meno spesa e meno lavoro potrei mettere gli interruttori IoT vimar originali, che era la seconda idea che mi è venuta

Però dovrebbe esistere un unico modulo centrale, da sostituire o aggiungere all’impianto esistente, che mi permetta di esporre in HA l’intero sistema tapparella/pulsante. Qualcuno ha fatto un lavoro simile? Consigli?
Grazie

**Reply by paolinor88** [reply]
Aggiungo che le mie tapparelle hanno anche le lamelle orientabili e non so se questa funzione è supportata dallo Shelly

**Reply by Buff81** [reply]
> * il costo mi frena un po’. 45€ a Shelly x 7 tapparelle……

perchè 45€? Su amazon vendono a 44€ la coppia, 22€ a tapparella. E sul sito shellyitalia a 19.99 il singolo + s.s.

**Reply by paolinor88** [reply]
Ah non avevo visto bene il prezzo. Ok, così già meglio…. Ma rimane il problema del bus centralizzato

**Reply by essegi65** [reply]
Se pensi di sostituire tutti gli attuatori il lavoro del BUS (e tanto altro ) lo puoi fare con gli script e/o automazioni di HA, in questo senso non avrai più limiti se non la tua fantasia. Per le lamelle sono orientabili elettricamente?

**Reply by Daniele7611** [reply]
ciao, alla fine come hai risolto?

**Comment by tempod** [comment]
personalmente non sostituirei mai un sistema cablato con uno wireless. Il componente centrale che espone tutto in HA che cerchi è un web server 01945 o 01946 ( a seconda di quanti dispositivi hai o vuoi montare) e userei questa integrazione:

https://community.home-assistant.io/t/vimar-by-me-by-web-integration/202073

ovviamente la spesa è molto diversa. ma anche la qualità e l'affidabilità non sono paragonabili

**Comment by tempod** [comment]
Una precisazione. Se scegli la strada di prendere il web server metti in conto o l'aiuto di un professionista o l'acquisto delle interfacce di programmazione 01993 e relativo 14329 (o quello adatto alla tua serie) per la configurazione.
